packages/coq-of-ocaml/coq-of-ocaml.2.5.3+4.14/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/formal-land/coq-of-ocaml"
+dev-repo: "git+https://github.com/formal-land/coq-of-ocaml.git"
+bug-reports: "https://github.com/formal-land/coq-of-ocaml/issues"
+authors: ["Guillaume Claret"]
+license: "MIT"
+build: [
+  ["sh" "-c" "cd proofs && ./configure.sh"] {coq:installed}
+  [make "-C" "proofs" "-j%{jobs}%"] {coq:installed}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  [make "-C" "proofs" "install"] {coq:installed}
+]
+depends: [
+  "angstrom" {>= "0.15.0"}
+  "csexp"
```

Comment:
There seems to be a lower-bound constraint missing here:
[lower-bounds (failed: Unbound type constructor Csexp.t)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5114feaf36e3735c6a5def87485d58a854d4077e/variant/compilers,4.14,coq-of-ocaml.2.5.3+4.14,lower-bounds)

I'm not sure what exact version you need, but the downgrade which reveals the error is:
```
downgrade csexp              1.5.1 to 1.0.0      [required by coq-of-ocaml]
```

Reply:
OK, thanks! I added a lower bound constraint.